pontoon/translations/views.py

```diff
@@ -19,9 +19,27 @@
 )
 from pontoon.checks.libraries import run_checks
 from pontoon.checks.utils import are_blocking_checks
+from pontoon.messaging.notifications import send_badge_notification
 from pontoon.translations import forms
 
 
+def _add_badge_data(response_data, user, badge_name, stat_count):
+    if badge_name == "Review Master Badge":
+        badge_level = settings.BADGES_REVIEW_THRESHOLDS.index(stat_count) + 1
+    else:
+        badge_level = settings.BADGES_TRANSLATION_THRESHOLDS.index(stat_count) + 1
```

Comment:
Seems like the threshold should be passed as a parameter to `_add_badge_data()`.